core-rust/core-api-server/src/core_api/conversions/substates/type_info_module.rs

```diff
@@ -3,8 +3,26 @@ use super::*;
 use crate::core_api::models;
 
 use radix_engine::types::*;
+use radix_engine::vm::VmBoot;
 use radix_engine_queries::typed_substate_layout::*;
 
+pub fn to_api_vm_boot_substate(
+    _context: &MappingContext,
+    _state_mapping_lookups: &StateMappingLookups,
+    substate: &VmBoot,
+) -> Result<models::Substate, MappingError> {
+    let substate = match substate {
```

Comment:
Very minor/picky, but for consistency, this is referred to as `value` rather than the substate.

For consistency, you could also create a `locked_versioned_system_field_substate` macro, but that's possibly overkill. The macro does prevent inconsistent naming though.